test/stripe/util_test.exs

```diff
@@ -34,6 +34,24 @@ defmodule Stripe.UtilTest do
       assert object_name_to_module("transfer") == Stripe.Transfer
       assert object_name_to_module("transfer_reversal") == Stripe.TransferReversal
       assert object_name_to_module("token") == Stripe.Token
+
+      assert object_name_to_module("billing_portal.session") == Stripe.BillingPortal.Session
```

Comment:
These become a bit extra, but I figure that I'd add manual tests for any of the handwritten functions I removed